trie-db/src/fatdb.rs

```diff
@@ -109,6 +109,22 @@ where
 	trie: &'db TrieDB<'db, 'cache, L>,
 }
 
+/// Double ended iterator over inserted pairs of key values.
+pub struct FatDBDoubleEndedIterator<'db, 'cache, L>
```

Comment:
I should have mention earlier, but fatdb iterator and fatdb generally is a bit dead code (old ethereum thing).
So maybe it is better not to have it (I know that it was remove in some substrate specific trie branches), but no harm having it.

Reply:
I think I'll just leave it if it isn't a big deal. If you insist, I will remove it.